src/main/target/REVO/AIRBOTF4SD.mk

```diff
@@ -0,0 +1,11 @@
+FEATURES       += VCP SDCARD
```

Comment:
For easier maintainability (all target config in 3 files), all of this should go into `target.mk`.